web/src/components/Table/Table.tsx

```diff
@@ -0,0 +1,434 @@
+            } else if (config.unique_column_values[column] <= 10) {
+              var choosenValues: any = []
+
+              filters[column].split(',').map((value: any) => {
+                if (value == '') return;
+                choosenValues.push(value)
+              })
+
+              return choosenValues.some((value: string) => cellValue.includes(value));
+            }
+          } else {
+            const filterValue = filters[column];
+            return cellValue.includes(filterValue);
+          }
+        });
+      });
+    }
+    
+    if (sortConfig) {
+      processedData.sort((a, b) => {
+        const columnIndex = config.columns.indexOf(sortConfig.key);
+        if (a[columnIndex] < b[columnIndex]) {
+          return sortConfig.direction === 'ascending' ? -1 : 1;
+        }
+        if (a[columnIndex] > b[columnIndex]) {
+          return sortConfig.direction === 'ascending' ? 1 : -1;
+        }
+        return 0;
+      });
+    }
+
+    return processedData;
+  }, [data, filters, sortConfig, config.columns]);
+
+  const currentRows = React.useMemo(() => {
+    return processData.slice(indexOfFirstRow, indexOfLastRow);
+  }, [processData, indexOfFirstRow, indexOfLastRow]);
+
+  const handleSort = (key: keyof TableRowProps, direction: 'ascending' | 'descending') => {
+    setSortConfig(prevConfig => {
+      if (prevConfig && prevConfig.key === key ) {
+        if (prevConfig.direction === direction) {
+          return null;
+        }
+      }
+      return { key, direction };
+    });
+  };
+
+  
+  const handleHideColumn = (column: string) => {
+    hideColumn(column);
+  };
+
+  const handleEmbedHistogram = (index: string) => {
+    showHistogram(index)
+  }
+
+  const handleOpenFilterPopup = (column: string, anchor: HTMLDivElement) => {
+    if (config.displayed_numeric_columns.includes(column)) {
+      setFilterType('brush')
+    } else if (config.unique_column_values[column]) {
+      if (config.unique_column_values[column] > 10) {
+        setFilterType('string')
+      } else if (config.unique_column_values[column] <= 10) {
+        setFilterType('tick')
+      }
+    } else {
+      setFilterType('string')
+    }
+    setFilterColumn(column);
+    setAnchorRefs(prevRefs => ({ ...prevRefs, [column]: anchor }));
+    setShowFilterPopup(true);
+  };
+
+  const handleApplyFilter = (value: string) => {
+    setFilters((prevFilters: any) => ({ ...prevFilters, [filterColumn || '']: value }));
+  };
+  const handleCloseFilterPopup = () => {
+    setShowFilterPopup(false);
+    setFilterColumn(null);
+  };
+
+  return (
+    <div>
+      <table id="table">
+        <thead>
+          <tr>
+            <th></th>
+            {showLineNumbers && (
+              <th></th>
+            )}
+            {visibleColumns.map((key: any) => (
+              <th key={key}>
+                <div className="th-inner">
+                  {key.charAt(0).toUpperCase() + key.slice(1).replace('_', ' ')}
```

Comment:
I don't think we always want to capitalize the first letter of a title. Also is the `label` keyword already supported? https://datavzrd.github.io/docs/configuration.html#columns